SeaPublicWebsite/Controllers/CookieController.cs

```diff
@@ -34,6 +34,7 @@ public IActionResult CookieSettings_Post(CookieSettingsViewModel viewModel)
         var cookieSettings = new CookieSettings
         {
             Version = cookieService.Configuration.CurrentCookieMessageVersion,
+            ConfirmationShown = true,
```

Comment:
Should this be false? Just because they are on the confirmation page doesn't mean that they have seen the confirmation banner?

Reply:
If they reach this it means they submitted their cookie preferences, which means they consented, so we don't need a banner anymore.

Reply:
We still need to tell them that they succeeded in setting their preference. However if you save the settings on this page there is a different message at the top of the page that says "You’ve set your cookie preferences" so we don't need the banner, so this is OK.

Reply:
Yup! I've already added a confirmation message whenever an user saves their settings here.